src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -5901,6 +5901,11 @@ internal override IOperation VisitPointerIndirectionReference(IPointerIndirectio
             return new PointerIndirectionReferenceExpression(Visit(operation.Pointer), semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, IsImplicit(operation));
         }
 
+        public override IOperation VisitTranslatedQuery(ITranslatedQueryOperation operation, int? captureIdForResult)
+        {
+            return new TranslatedQueryExpression(Visit(operation.Operation), semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, IsImplicit(operation));
+        }
+
```

Comment:
I think there is no change here other than swapping the source position of two methods.